index.js

```diff
@@ -25,16 +26,35 @@ var getObject = function (detail) {
 	}
 	return obj;
 };
+var getBorderChars = function (borderStyle) {
+	var chars;
+	if(typeof borderStyle === 'string') {
+		chars = borderChars[borderStyle];
+		if (!chars) {
+			throw new TypeError('Invalid borderStyle ' + borderStyle);
+		}
+	} else {
+		['topLeft', 'topRight', 'bottomRight', 'bottomLeft', 'vertical', 'horizontal'].forEach(function (key) {
+			if(!borderStyle[key] || typeof borderStyle[key] !== 'string') {
+				throw new TypeError('Invalid borderStyle, missing or wrong valued key ' + key);
```

Comment:
Did I make the phrase `wrong valued` up? I am not a native tongue.

Other suggestions, @SamVerschueren?